examples/tritonserver/app/README.md

```diff
@@ -0,0 +1,12 @@
+# Triton server
+
+Triton server can be used to inference AI workloads using machine learning models. there are couple of referece AI workloads like fraud detection, Iris classification covered in the [repo](https://github.com/PDeXchange/ai-demos)
+Users can deploy AI workloads of their choice of model and configuration by supplying the trained model file(model.onnx) and configuration file (config.pbtxt) to http server to be used by Triton server when its run on a PIM partition.
+
+## Fraud detection usecase with ONNX runtime
```

Comment:
Mention this script will build the application examples given in https://github.com/PDeXchange/ai-demos.

Reply:
done